src/server/game/Maps/Map.h

```diff
@@ -31,6 +31,7 @@
 #include "DynamicTree.h"
 #include "GameObjectModel.h"
 #include "ObjectGuid.h"
+#include "Creature.h"
```

Comment:
you dont even need this here - for the future please carefully think about adding includes inside other headers